src/routes/docs/products/functions/runtimes/+page.markdoc

```diff
@@ -149,7 +149,7 @@ While still in beta, Appwrite Cloud has limited support for Cloud runtimes. As w
 ---
 * {% icon icon="go" size="l" /%}
 * [Go](https://hub.docker.com/r/openruntimes/go/tags)
-* `go-1.22`
+* `go-1.23`
```

Comment:
Also this may conflict with my current PR, which already addresses the Go versions in the docs https://github.com/appwrite/website/pull/1249

Reply:
reverted